scrapi/processing/cassandra.py

```diff
@@ -73,6 +73,48 @@ def different(self, old, new):
 
 
 @database.register_model
+class DocumentModelV2(models.Model):
+    '''
+    Defines the schema for a metadata document in cassandra
+
+    The schema contains denormalized raw document, denormalized
+    normalized (so sorry for the terminology clash) document, and
+    a list of version IDs that refer to previous versions of this
+    metadata.
+    '''
+    __table_name__ = 'documents_v2'
+
+    # Raw
+    source = columns.Text(primary_key=True, partition_key=True)
```

Comment:
Not really, I just put source first because I thought it made sense. Documents are grouped first by source, and then within source ordered by docID